src/main/java/com/spotinst/sdkjava/model/api/admin/organization/ApiUserGroupDetailsPolicies.java

```diff
@@ -0,0 +1,91 @@
+package com.spotinst.sdkjava.model.api.admin.organization;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiUserGroupDetailsPolicies implements IPartialUpdateEntity {
+    @JsonIgnore
+    private Set<String>  isSet;
+    private String       policyId;
+    private String       policyName;
+    private String       policyType;
+    private List<String> accountIds;
```

Comment:
Should it be accountIds or accountNames? As per documentation, it is accountNames